src/gui/tray/ActivityItem.qml

```diff
@@ -121,7 +111,7 @@ MouseArea {
             adjustedHeaderColor: root.adjustedHeaderColor
 
             onTriggerAction: activityModel.slotTriggerAction(model.index, actionIndex)
-            onShowReplyField: root.toggleReplyOptions()
+            onShowReplyField: root.isTalkReplyOptionVisible = true
```

Comment:
@claucambra I remember Camila's been fixing some issues and had to implement and use that `toggleReplyOptions()`. Please, make sure that you really need to remove it and that we won't have the regression.

Reply:
`toggleReplyOptions` is only ever called by this `onShowReplyField` signal handler and its contents only flip `isTalkReplyOptionVisible` to true -- but @camilasan if I am missing something here I am happy to revert